configuration/configuration.go

```diff
@@ -94,6 +99,7 @@ type Configuration struct {
 	RemoteGeth             bool
 	Port                   int
 	GethArguments          string
+	SkipGethAdminEnv       bool
```

Comment:
in the configuration struct, I would keep the `SkipGethAdmin` name since `configuration.SkipGethAdmin` reads better. Sorry about the confusion :(

Reply:
No problem and good spot! Fixed!